### PR TITLE
V2.1.0 rc docs part 1

### PR DESCRIPTION
**Pull request by roxy-dao**
Not all the changes in v2.1.0 done yet... Will go through it some more soon....

### CODE REVIEW

File: content/docs/replenishment/internal_orders.md [hunk 1]
```diff
@@ -165,15 +166,7 @@ When you add items (using a master list or not), the item is added to the order'
 | **AMC**                  | Average Monthly Consumption: how much stock your store uses each month on average (based on a configurable number of months, defaults to 3 months) |
 | **Target Stock**         | This is the stock you are aiming for. Calculated as: AMC x Target MOS MOS                                                                          |
 | **Suggested Quantity**   | How much stock mSupply suggests that your order. This is calculated as: (AMC x Target MOS) - SoH                                                   |
-| **Requested**            | This is set to zero by default. This is the quantity of units you are ordering from your supplier.                                                 |     |
-
-### Using Suggested Quantities
-
```
Comment (roxy-dao) [inline content/docs/replenishment/internal_orders.md:171]: Moved up... was being repeated

File: content/docs/coldchain/equipment.md [hunk 2]
```diff
@@ -76,17 +77,10 @@ Cold chain assets can be imported from a comma-separated-value (csv) file using
...
+This will open an import modal. Click on `Download a template` to download a csv
+template which can be used to import assets.
 
-<p align="center">
-    <img src="/docs/coldchain/images/equipment_import_modal.png" width="750" height="420">
-</p>
-
-An example template csv is available for download here:
-
-<p align="center">
```
Comment (roxy-dao) [inline content/docs/coldchain/equipment.md:87]: Removed and added above

File: content/docs/coldchain/equipment.md [hunk 3]
```diff
@@ -181,9 +175,13 @@ From the `Summary` tab, you can:
 
 <div class="tip">Multiple locations can be assigned to one asset</div>
 
-You can also view the functional status and edit the notes of the status log.
```
Comment (lache-melvin) [inline content/docs/coldchain/equipment.md:184]: Just above, it says there are 4 tabs available but there are 5 now 😁

File: content/docs/coldchain/monitoring.md [hunk 4]
```diff
@@ -42,7 +42,10 @@ The chart tab displays a chart showing all configured sensors. You can filter th
 
 To add a filter to the page, choose the required filter from the drop down, the date time filter will apply to all of the tabs in the monitoring section.
 
-<div class="note">By default, data from the last 24 hours is shown. Use the from date/time and to date/time fields to look at a longer period of time.</div>
+<div class="note">By default, data from the last 24 hours is shown. Use the from date/time and to date/time fields to look at a longer period of time.
```
Comment (lache-melvin) [inline content/docs/coldchain/monitoring.md:45]:
```suggestion
<div class="note">By default, data from the last 24 hours is shown. Use the <code>From date/time</code> and <code>To date/time</code> fields to look at a longer period of time.
```

Maybe? "Use the from" always reads as a typo for me..

File: content/docs/dispensary/patients.md [hunk 5]
```diff
@@ -14,33 +14,36 @@ toc = true
...
+| **Patient ID**    | The ID of the patient            |
+| **NUIC**          | National unique identifier code  |
+| **Created**       | The date the patient was created |
+| **First name**    | First name                       |
+| **Last name**     | Last name                        |
+| **Gender**        | Gender                           |
+| **Date of Birth** | The patient's date of birth      |
+| **Deceased**      | Whether the patient is deceased  |
 
 2. The list can display a fixed number of patients per page. On the bottom left corner, you can see how many patients are currently displayed on your screen.
```
Comment (lache-melvin) [inline content/docs/dispensary/patients.md:48]: Not related to your changes, but why are these next bits numbered... from 2??

File: content/docs/getting_started/central.md [hunk 6]
```diff
@@ -56,7 +56,11 @@ To configure your Open mSupply site to use a central server, there are a couple
 
 - In mSupply, create a new site with a store assigned (this can be a dummy store)
 - Tick the checkbox (as noted in the 'extra settings' link above)
-- Enter the server URL. Note that this URL is that of your new Open mSupply central server. This will be different to the usual Open mSupply server and to the mSupply server!
+- Enter the server URL. Note that this URL is that of your new Open mSupply
```
Comment (lache-melvin) [inline content/docs/getting_started/central.md:59]:
```suggestion
- Enter the server URL of your new Open mSupply
```

File: content/docs/getting_started/central.md [hunk 7]
```diff
@@ -56,7 +56,11 @@ To configure your Open mSupply site to use a central server, there are a couple
...
+  central server. This will be different to the usual Open mSupply server and to
+  the mSupply server!
+
+<div class="warning">Open mSupply central server can be configured and sync even if the URL is wrong since it's only the remote sites that need to use the Open mSupply central server URL to connect.</div>
```
Comment (lache-melvin) [inline content/docs/getting_started/central.md:63]:
```suggestion
<div class="warning">Open mSupply central server can be configured and synced successfully, even if this URL is wrong. This field is only used by remote sites, to know where to find the Open mSupply central server.</div>
```
🤷‍♀️

**Review by lache-melvin — APPROVED** [review]
TY 🙏

File: content/docs/dispensary/patients.md (hunk 5 above)
Comment (roxy-dao) [reply content/docs/dispensary/patients.md:48]: Cause missed the 1 above XD

**Pull request merged**